debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -749,6 +778,14 @@ private void assertRecordsAfterInsert(int expectedCount, int...pks) throws Inter
         IntStream.range(0, expectedCountPerSchema).forEach(i -> VerifyRecord.isValidInsert(recordsForTopicS2.remove(0), PK_FIELD, pks[i]));
     }
 
+    protected void assertSourceInfoTransactionTimestamp(SourceRecord record, Long ts_usec, Long tolerance_usec) {
+        assertTrue(record.value() instanceof Struct);
+        Struct source = ((Struct) record.value()).getStruct("source");
+        // 1 minute difference is okay
+        System.out.println("TS_USEC\t" + source.getInt64("ts_usec"));
```

Comment:
Ooops. I forgot to remove this. Moving this into the `assertTrue` message would be better.

Reply:
Will take care of it.